libs/wingc/src/lifting.rs

```diff
@@ -281,8 +281,30 @@ impl<'a> Visit<'a> for LiftVisitor<'a> {
 		// CAPTURE
 		if self.should_capture_type(&node) {
 			// jsify the type so we can get the preflight code
-			let code = self.jsify_udt(&node);
-
+			let current_env = self.ctx.current_env().expect("an env");
+			let code = if let Some(SymbolKind::Namespace(root_namespace)) = current_env.lookup(&node.root, None) {
+				match &root_namespace.kind {
+					// types in wing files already implement a helper to lift types
+					NamespaceKind::FileModule => self.jsify_udt(&node),
+
+					NamespaceKind::JSII { fqn } => {
+						let mut fqn = fqn.clone();
+						if root_namespace.name != fqn {
+							fqn = format!("{}/{}", fqn, root_namespace.name);
+						};
+						let type_path = node
+							.fields
+							.iter()
+							.map(|f| f.name.clone())
+							.collect::<Vec<String>>()
+							.join(".");
+						let udt = self.jsify_udt(&node);
+						format!("$stdlib.core.toLiftableModuleType({udt}, \"{fqn}\", \"{type_path}\")",)
```

Comment:
Since `toLiftableModuleType` returns the input type if it has a `toInflightType` method then why can't we call it for **all** UDTs (`node`s)? 
Can we change the code here to always calculate `fqn` and `type_path` then always call `toLiftableModuleType`? Then we don't need the `NamespaceKind`. 
Finally, perhaps, move this into `jsify_udt` to keep our JS generating code under the `jsify_*` prefix.

Reply:
> Since toLiftableModuleType returns the input type if it has a toInflightType method then why can't we call it for all UDTs (nodes)?

Not all UDTs have a fqn representation right now. To be clear, fqn here must refer to something `require`-able. For namespaces in JSII libs this could be a npm package name or path, whereas for wing type it would have to be a path to a js file. I'm not sure we currently have a way to get the path to `require` a wing namespace, and it's not necessary since wing types always have toInflightType.

> move this into jsify_udt to keep our JS generating code under the jsify_* prefix.

Can do